roles/openshift_logging/defaults/main.yml

```diff
@@ -72,7 +72,7 @@ openshift_logging_fluentd_nodeselector: "{{ openshift_hosted_logging_fluentd_nod
 openshift_logging_fluentd_cpu_limit: 100m
 openshift_logging_fluentd_memory_limit: 512Mi
 openshift_logging_fluentd_es_copy: false
-openshift_logging_fluentd_use_journal: "{{ openshift_hosted_logging_use_journal | default('') }}"
+openshift_logging_fluentd_use_journal: "{{ openshift_hosted_logging_use_journal if openshift_hosted_logging_use_journal is defined else (docker_log_driver|default('') == 'journald') | ternary(True, False) }}"
```

Comment:
@ewolinetz https://github.com/openshift/openshift-ansible/blob/97cf14a03d2b392e2355478c74882d7be9f653d2/playbooks/common/openshift-cluster/initialize_oo_option_facts.yml#L20 and https://github.com/openshift/openshift-ansible/blob/bc5eb65f9bcd4147130e062ef2e4499145811c90/roles/openshift_docker_facts/tasks/main.yml#L29
the latter is docker.log_driver - I'm not sure where/how this is set, but there is a reference to it in the facts module https://github.com/openshift/openshift-ansible/blob/bc5eb65f9bcd4147130e062ef2e4499145811c90/roles/openshift_facts/library/openshift_facts.py#L65